easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2022.05-foss-2022a.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2022.05'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+builddependencies = [
+    ('hypothesis', '6.46.7'),
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('pybind11', '2.9.2'),  # required by scipy
+]
+
+use_pip = True
+
+# order is important!
+exts_list = [
+    ('numpy', '1.22.3', {
```

Comment:
@branfosj Wasn't the plan here to go with latest `numpy` (1.30)?

That does require a more recent `Cython` than we have included with `Python`, but that could be added as a build dependency for `SciPy-bundle`?

I guess we can also make that change in a separate PR once this one is merged...

Reply:
@akesandgren suggested going with `numpy` `1.23.0`, but, yes, it is more complicated due to the need for an updated `Cython`. It would be best to do that separately to this PR.